fdbserver/ptxn/test/RealTLog.actor.cpp

```diff
@@ -90,19 +90,18 @@ ACTOR Future<Void> getTLogCreateActor(std::shared_ptr<TestDriverContext> pTestDr
 	Promise<Void> recovery;
 
 	// construct tLog.
-	state Future<Void> tl = tLog(pTLogContext->persistentData,
-	                             pTLogContext->persistentQueue,
-	                             dbInfoRef,
-	                             localities,
-	                             promiseStream,
-	                             pTLogContext->tLogID,
-	                             pTLogDriverContext->workerID,
-	                             false, /* restoreFromDisk */
-	                             oldLog,
-	                             recovery,
-	                             pTLogDriverContext->diskQueueBasename,
-	                             isDegraded,
-	                             activeSharedTLog);
+	state Future<Void> tl = ::ptxn::tLog({ { pTLogContext->persistentData, pTLogContext->persistentQueue } },
```

Comment:
Maybe revert this change and let Dan create new tests for `::ptxn::tLog`.

Reply:
Done